spec/client/services/adminDashDataService.spec.js

```diff
@@ -420,4 +424,94 @@ describe("adminDashDataService", function () {
             }
         );
     });
+
+    describe("addSpeaker", function () {
```

Comment:
The test descriptions for all of these are still talking about the blocked users endpoint